taskfile.yaml

```diff
@@ -14,6 +15,13 @@ tasks:
     cmds:
       - "rm -rf '{{.G_OUTPUT_DIR}}'"
 
+  init:
```

Comment:
nit: `init` should be after `test` since it is an `internal` task
(https://docs.yscope.com/dev-guide/contrib-guides-taskfiles.html#task-ordering)

I was skimming through this PR since something I'm working on will need to merge with it.